examples/cstg-euid/html/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>EUID Publisher Client-Side Integration Example using UID2 JavsScript SDK</title>
+    <link rel="stylesheet" type="text/css" href="/stylesheets/app.css" />
+    <link rel="shortcut icon" href="/images/favicon.png" />
+    <script defer src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script defer src="{{ EUID_JS_SDK_URL }}"></script>
+    <script>
+      const clientSideIdentityOptions = {
+        subscriptionId: '{{ SUBSCRIPTION_ID }}',
+        serverPublicKey: '{{ SERVER_PUBLIC_KEY }}',
```

Comment:
You'll need EUID values for these (see my comment on the Prebid.js example PR) but they can just be included here - they don't need to be server-side config values like this.